src/app/styles/device.js

```diff
@@ -0,0 +1,12 @@
+import { breakpoints } from "./theme/ui/index";
+
+const device = {
+  mobileS: `(min-width: ${breakpoints.xs})`,
+  mobileM: `(min-width: ${breakpoints.sm})`,
+  tablet: `(min-width: ${breakpoints.md})`,
+  desktop: `(min-width: ${breakpoints.lg})`,
+  desktopL: `(min-width: ${breakpoints.xl})`,
+  desktopHD: `(min-width: ${breakpoints.xxl})`,
+};
```

Comment:
I would name them the same as breakpoints, mobileS = xs, mobileM = sm, etc. Plus additional ideas can be found here: https://www.codevertiser.com/styled-components-media-queries/